.gitignore

```diff
@@ -85,6 +85,7 @@ installdir/
 ######
 .doit.db.dat
 .doit.db.dir
+.doit.db.db
```

Comment:
Interesting, on my machine I see `.doit.db` that is not being ignored--you're getting the repeated extension on your end? I guess we can add yours and mine.

I noticed that the one I see is also in the `pydoit` `.gitginore` proper: https://github.com/pydoit/doit/blob/master/.gitignore#L2

Reply:
Yea I get `.doit.db.db` although it looks like the doit docs only mention `.doit.db`.

I updated the PR by adding `.doit.db` as well: [`fa56596` (#15999)](https://github.com/scipy/scipy/pull/15999/commits/fa565963c82663ce5c628824f6371115f324c228)

Reply:
doit internally uses stdlib `dbm` [1]. Which implementation (and extension applied) depends on the default implementation in your installation. Unfortunately the extension added (or not) to the files is inconsistent... 

[1] https://docs.python.org/3/library/dbm.html